qualtran/bloqs/multi_control_multi_target_pauli_test.py

```diff
@@ -41,3 +45,39 @@ def test_t_complexity_mcp(num_controls: int, pauli: cirq.Pauli, cv: int):
     gate = MultiControlPauli([cv] * num_controls, target_gate=pauli)
     assert_valid_bloq_decomposition(gate)
     assert_decompose_is_consistent_with_t_complexity(gate)
+
+
+@pytest.mark.parametrize("cvs", [(), (0), (1, 0), (1, 1, 1), (1, 0, 1, 0)])
+def test_multi_control_x(cvs):
+    bloq = MultiControlX(cvs=cvs)
+    assert_valid_bloq_decomposition(bloq=bloq)
+
+
+@pytest.mark.parametrize(
+    "cvs,x,ctrl",
```

Comment:
this doesn't match the order of the names you pass in to the actual function (??)